changelog.json

```diff
@@ -1,5 +1,9 @@
 {
   "releases": {
+    "1.7.0-beta4": [
+      "[Fixed] Ensure manual resolution choices are in sync with rebase flow - #7255",
+      "[Fixed] Show \"force push\" label after rebase which requires it - #4976 #7138 #7231"
```

Comment:
```suggestion
      "[Fixed] Menu items don't display the expected verbiage for force push and removing a repository - #4976 #7138"
```

Reply:
@iAmWillShepherd were we going to take in this change?